docs/source/index.rst

```diff
@@ -41,34 +55,33 @@ The following commands will download and build UCX v1.6 :doc:`release <download>
     $ ./contrib/configure-release --prefix=$PWD/install
     $ make -j8 install
 
+    $ gcc test/examples/ucp_client_server.c -lucp -lucs -o ucp_client_server \
+          -Iinstall/include -Linstall/lib
+    $ export LD_LIBRARY_PATH=$PWD/instal/lib
```

Comment:
do we really need LD_xxx ?

Reply:
yes, since in this example ucx is installed to local path (to not require root user)